src/node/Main.hs

```diff
@@ -181,8 +181,8 @@ processUserSecret
 processUserSecret args@Args {..} userSecret = case backupPhrase of
     Nothing -> updateUserSecretVSS args userSecret >>= userSecretWithGenesisKey args
     Just ph -> do
-        let (sk, vss) = keysFromPhrase ph
-            us = userSecret & usPrimKey .~ Just sk & usVss .~ Just vss
+        (sk, vss) <- either (\msg -> fail $ "Key creation from phrase failed: " <> show msg) pure $ keysFromPhrase ph
```

Comment:
Maybe wrap `keysFromPharse` to next line.
Or, @ChShersh recently told me about `whenLeftM`-like functions, it seems to fit